src/components/CreatePage.js

```diff
@@ -118,9 +118,16 @@ class CreatePage extends Component {
     return (
       <div className={classnames(styles.container, 'container')}>
         <section>
-          <div>{this.renderContent()}</div>
+           <div>{this.renderContent()}</div>
         </section>
-      </div>
+        {/* <div className={styles.appBarBottom}></div> */}
+        <BottomAppBar 
+          content={<Button 
+            className={styles.submitButton} 
+            onClick={this.handleSubmit}
+            raised>Submit</Button>} 
+          />
```

Comment:
Any reason why this `content` prop isn't called `children` instead? That'll allow you to do something like the suggested change. It's not formatted correctly but if you run Prettier it'll auto format for us.

```suggestion
        <BottomAppBar>
          <Button 
            className={styles.submitButton} 
            onClick={this.handleSubmit}
            raised>Submit</Button>
        </BottomAppBar>
```

Reply:
yeah next commit i made it props.children alr